extra_data/reader.py

```diff
@@ -289,18 +292,39 @@ def __init__(self, files, selection=None, train_ids=None, ctx_closes=False):
             train_ids = sorted(set().union(*(f.train_ids for f in files)))
         self.train_ids = train_ids
 
+    @staticmethod
+    def _open_file(path, cache_info=None):
+        try:
+            fa = FileAccess(path, _cache_info=cache_info)
+        except Exception as e:
+            print("Skipping file", path, file=sys.stderr)
+            print("  (error was: {})".format(e), file=sys.stderr)
```

Comment:
It might be necessary to transmit the error message back to the main process and show it there - both for the tests, and because I'm not sure printing from a multiprocessing child process would show up in Jupyter.